.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,47 @@
-# Summary
+# Summary [Required]
+<!-- Provide a concise description (2-4 sentences) of what changes this PR implements and why -->
 
-# Demo
+# Demo [Required for UI changes]
+<!-- Add screenshots or link to screen recording demonstrating the changes -->
+<!-- Include both light/dark mode and different device sizes if it matters -->
 
-Add screenshots or flow video
+## Release Notes
+<!-- 
+Release notes with appropriate tags will be included in automatically generated release notes.
+Use the following guidelines to determine if you need release notes:
 
-# Release notes
+1. Changes targeting feature branches: Generally NO release notes needed
+2. Changes targeting `develop` branch: Include release notes if merchants should be informed
+3. Any user-facing changes (UI, workflows, public API changes): Include release notes
+-->
 
-<fixed> 
-- Fixed a bug.
+### Use appropriate tags for different types of changes:
+<fixed>
+- Fixed [specific issue] that caused [specific impact]
 </fixed>
 
 <new>
-- Added new feature.
+- Added ability to [specific capability] which enables [specific benefit]
 </new>
 
-# Ticket
+<changed>
+- Modified [specific behavior] to now [specific new behavior]
+</changed>
 
+## Technical Information [Optional]
+<!-- Include any technical details that would help reviewers understand implementation choices -->
+
+## Testing Instructions [Optional]
+<!-- Step-by-step instructions for reviewers to verify your changes work as expected -->
+
+## Ticket [Required]
 <ticket>
 COIOS-
 </ticket>
+
+## Checklist [Required]
+<!-- Mark completed items with an [x] -->
+- [ ] Tested changes locally
+- [ ] Added/updated unit tests
+- [ ] Verified against acceptance criteria
+- [ ] PR title follows conventional commit format
```

Comment:
Can we add an example for commit message format ?

Reply:
I tried to come up with a good system that encompasses our style and I found we have no style.

Patterns I found:

Feature - Feature part
Ticket - feature
random things following no specific convention

Should I just remove this requirement for now?

Reply:
Removed for now
In my ideal world PR title and description are filled with the help of AI bot so we don't spent even a second on this